tests/test_data_frame.py

```diff
@@ -603,5 +603,23 @@ def test_complex_write_attr_access(self):
         assertDataFrameEqualsPandas(df.p, pd.DataFrame(base_df.p))
         assertDataFrameEqualsPandas(df_1, pd.DataFrame(base_df_1))
 
+    def test_drop_duplicates(self):
```

Comment:
This only tests `drop_duplicates` on SQLite. Let's add corresponding tests in `test_pandas_execution.py` too.

Also, maybe check `df.sql()` in line with other tests in this file?

Reply:
I'll add the same tests to `test_pandas_execution.py`, but just for future simplicity is there a reason to have them be separate files? Could we just have the test suite run once with offloading and once without?

Reply:
In retrospect, it was probably a poor test design choice. We should do that. I'll make an issue!